packages/dds/tree/src/test/simple-tree/api/create.spec.ts

```diff
@@ -31,4 +39,23 @@ describe("simple-tree create", () => {
 		const canvas2 = new Canvas({ stuff: [] });
 		assert.deepEqual(canvas1, canvas2);
 	});
+
+	describe("cursorFromInsertable", () => {
```

Comment:
Moved from src/test/simple-tree/toMapTree.spec.ts (for some reason this was inside of toMapTree's schema validation section)